templates/portfolios/portfolios_table.html

```diff
@@ -29,15 +29,13 @@
     {% if "can_grant_portfolio_owner_permission" in perms %}
     <div class="col-md-2">Owner</div>
         <div class="col-md-2">
-            <span class="glyphicon glyphicon-pencil"></span>
-            <a id="new-project" class="portfolio-project-link" onclick="show_manage_portfolio_modal('{{portfolio.id}}', '{{ portfolio.title }}'); return false;" href="#">Manage Portfolio</a>
+            <a id="" class="portfolio-project-link" href="/portfolios/{{ portfolio.id }}/edit" title="Edit portfolio"><span class="glyphicon glyphicon-pencil"></span></a>
         </div>
     {% elif "change_portfolio" in perms%}
     <div class="col-md-2">Portfolio Member</div>
     <div class="col-md-2">
         <div class="col-md-2">
```

Comment:
Potentially duplicate? Unnecessary nesting?

Reply:
A portfolio member doesn't have certain permissions that a owner does

Reply:
I guess what I'm saying here is that they both have access to the same link (the "a" tag is basically duplicated between the 'if' and 'else' statement), so maybe that part should be pulled out of the 'if' statement and just have the "Owner"/"Portfolio Manager" text in the 'if' statement (i.e. the part that is different).

Reply:
Here I was just referencing the nested divs with the "col-md-2" class. Seems like one is a duplicate, based on surrounding html with a similar pattern.